### PR TITLE
[HEIDELPAYSUPPORT-110] Fix sales channel specific config for webhooks

### PR DESCRIPTION
**Pull request by ngoedde**

### CODE REVIEW

**Review by sixer1182 — CHANGES_REQUESTED** [review]
Please update the changelog with a description of this fix.

File: composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "unzerdev/shopware6",
     "description": "Unzer payment integration for Shopware 6",
-    "version": "2.0.1",
+    "version": "2.0.22,
```
Comment (sixer1182) [inline composer.json:4]: Here is something wrong

**Review by sixer1182 — CHANGES_REQUESTED** [review]
fix version in composer.json

**Review by sixer1182 — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (JoeKre) [reply composer.json]: Fixed

**Pull request closed without merging**